corpdir-srv-sync/src/main/java/com/github/bordertech/corpdir/sync/service/SynchronisationService.java

```diff
@@ -0,0 +1,18 @@
+package com.github.bordertech.corpdir.sync.service;
+
+import com.github.bordertech.corpdir.api.common.ApiIdObject;
+import java.io.Serializable;
+
+/**
+ * Sync service interface.
+ *
+ * @param <T> the criteria type
+ * @param <U> the response type
+ * @author aswinkandula
+ * @since 1.0.0
+ */
+
+public interface SynchronisationService<T extends Serializable, U extends ApiIdObject> {
+
+	public U sync(T criteria);

```

Comment:
What is criteria used for? If not used it could be removed to make the interface have less "noise".

Reply:
What do you think of introducing a `SyncException` that this method can throw?